src/actions/sendContactForm.ts

```diff
@@ -0,0 +1,52 @@
+"use server";
+
+import nodemailer from "nodemailer";
+
+interface SendEmailProps {
+  name: string;
+  email: string;
+  subject: string;
+  message: string;
+  token: string;
+}
+
+export async function sendContactForm({ name, email, subject, message, token }: SendEmailProps) {
```

Comment:
I suppose, we need runtime validation here. Maybe Zod or something like that. In addition, we can reuse validation schema on frontend

Reply:
Updated. Used `zod` for validation on UI and BE via sharing the schema